firebase-appdistribution/src/test/java/com/google/firebase/appdistribution/AabUpdaterTest.java

```diff
@@ -100,7 +95,7 @@ public void updateAppTask_whenOpenConnectionFails_setsNetworkFailure()
     when(mockHttpsUrlConnectionFactory.openConnection(TEST_URL)).thenThrow(caughtException);
 
     UpdateTask updateTask = aabUpdater.updateAab(TEST_RELEASE_NEWER_AAB_INTERNAL);
-    testExecutor.awaitTermination(1, TimeUnit.SECONDS);
+    testExecutor.awaitTermination(100, TimeUnit.MILLISECONDS);
```

Comment:
Was there some timeout issue that you changed the waiting from 1 second to 100 ms?

Reply:
Just to reduce the maximum waiting time of running these tests. 1 second seems like overkill.